vasern/src/core/vasern/Document.js

```diff
@@ -169,7 +169,7 @@ export default class Document {
   // @input: a valid record
   insert(records, save = true) {
     if (!records) {
-      return false;
+      return false; // Question how about just returning an empty array here? Would be great for TS
```

Comment:
Yeah, `return false` here isn't really help. Calling insert without a record is a developer mistake.
How about throw an error so the developer know they was calling `insert` without any record?

Reply:
That would certainly be a better approach than just returning false. And hopefully saves debugging time for developers who made that mistake and try to figure out why they only get false as a result.

Reply:
Great, I will make an update which will throw an error!